cmd/migrate/migrate.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/ledgerwatch/bolt"
+	"github.com/ledgerwatch/turbo-geth/common/dbutils"
+)
+
+func encodeToCBOR(buf []byte) ([]byte, error) {
+	acc, err := decodeRLP(buf)
+
+	if err != nil {
+		return nil, err
+	}
+
+	res := make([]byte, acc.EncodingLengthForStorage())
+	acc.EncodeForStorage(res)
+	return res, nil
+}
+
+func convertDatabaseToCBOR(db *bolt.DB, maxTxOperations uint) error {
+	var lastKey []byte = nil
+	var done bool = false
+	for !done {
+		var i uint
+		err := db.Update(func(readTx *bolt.Tx) error {
+			var accountBucket *bolt.Bucket = readTx.Bucket(dbutils.AccountsBucket)
+			err := accountBucket.ForEach(func(k, v []byte) error {
```

Comment:
You cannot use `ForEach` anymore, because it starts from the beginning of the bucket. Instead, you should use a cursor:
````
c := accountBucket.Cursor()
for k, v := c.Seek(lastKey); k != nil; k, v = c.Next() {
}
````